src/styles/blocks/card.scss

```
@@ -0,0 +1,120 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid ...;
```